tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -147,16 +148,18 @@ sub run_test {
 
     }
 
+    set_var("SRIOV_NETWORK_CARD_PCI_PASSSHTROUGH", 0);    #turn off the flag in case of affecting other tests
```

Comment:
But if you need to flag which test calls a function, a variable like SRIOV_NETWORK_CARD_PCI_PASSSHTROUGH is not a good choice, because it also depends on test execution order. If other test run before pci passthrough, then detecting SRIOV_NETWORK_CARD_PCI_PASSSHTROUGH becomes useless. The correct way is to use "caller", for exmaple,

sub myFunc{
my ($package, $filename, $line) = caller;
print $filename;
}

This way enables you to differentiate which module or file calls shared function.

Reply:
I will think it over and consider improving it in the next version soon.